### PR TITLE
gh-pages setup

### PR DESCRIPTION
**Pull request by noelmarkham**
SBT setup for pushing to gh-pages branch. It does work, it's already deployed to https://47deg.github.io/scalacheck-datetime/

@rafaparadela @juanpedromoreno would you be able to take a quick look please?

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -21,7 +24,14 @@ lazy val tutDirectoriesSettings = Seq(
...
 
-lazy val docsSettings = buildSettings ++ dependencies ++ tutSettings ++ tutDirectoriesSettings
+lazy val noPublishSettings = Seq(
+  publish := (),
+  publishLocal := (),
+  publishArtifact := false)
+
+lazy val ghpagesSettings = ghpages.settings ++ Seq(git.remoteRepo := "git@github.com:47deg/scalacheck-datetime.git")
+
+lazy val docsSettings = buildSettings ++ dependencies ++ tutSettings ++ tutDirectoriesSettings ++ noPublishSettings ++ ghpagesSettings
```
Comment (juanpedromoreno) [inline build.sbt]: Very Long line, maybe you could:

```
lazy val ghpagesSettings = 
  buildSettings ++ 
  dependencies ++ 
  ...
```

File: build.sbt [hunk 2]
```diff
@@ -21,7 +24,14 @@ lazy val tutDirectoriesSettings = Seq(
   tutTargetDirectory := sourceDirectory.value / "jekyll"
 )
 
-lazy val docsSettings = buildSettings ++ dependencies ++ tutSettings ++ tutDirectoriesSettings
+lazy val noPublishSettings = Seq(
+  publish := (),
+  publishLocal := (),
+  publishArtifact := false)
+
+lazy val ghpagesSettings = ghpages.settings ++ Seq(git.remoteRepo := "git@github.com:47deg/scalacheck-datetime.git")
```
Comment (juanpedromoreno) [inline build.sbt]: Long line here too.

File: build.sbt [hunk 3]
```diff
@@ -21,7 +24,14 @@ lazy val tutDirectoriesSettings = Seq(
   tutTargetDirectory := sourceDirectory.value / "jekyll"
 )
 
-lazy val docsSettings = buildSettings ++ dependencies ++ tutSettings ++ tutDirectoriesSettings
+lazy val noPublishSettings = Seq(
```
Comment (juanpedromoreno) [inline build.sbt]: What is the purpose of this setting list?

**Comment by juanpedromoreno** [comment]
Regarding the deployment, I don't know if the text logo could be fixed through CSS or similar.. Any thoughts @rafaparadela ?

screen shot 2016-07-22 at 15 05 29
screen shot 2016-07-22 at 15 05 37

File: build.sbt (hunk 3 above)
Comment (noelmarkham) [inline build.sbt]: I'd followed the instructions at https://github.com/47deg/microsites/blob/master/jekyll/docs.md#step-1-create-a-new-sbt-project-called-docs - perhaps I don't need it, I'll check and remove it.

**Comment by rafaparadela** [comment]
LGTM. But yes, please invite to Isra to this repo in order to refine some design issues.

Other recomendation: Edit your `jekyll/config.yml` in order to a highlight_theme. One example: `highlight_theme: tomorrow` (https://github.com/47deg/github4s/blob/master/docs/src/jekyll/_config.yml#L6)

You can choose your favourite theme from those: https://highlightjs.org/static/demo/

**Comment by noelmarkham** [comment]
Hi @israelperezglez - just wondering if you had any input on Juan Pedro and Rafa's comments above?

**Comment by noelmarkham** [comment]
Have added a highlight, will merge but keep the PR open for input from @israelperezglez.

**Pull request merged**